codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -206,12 +206,9 @@ public void sendFlowControlled(ChannelHandlerContext ctx, Http2Stream stream, Fl
             frame.error(t);
             return;
         }
+        // Write data to channel but don't flush as more writes may be coming and we want to
```

Comment:
Do you want to remove all calls to flush from the `DefaultHttp2RemoteFlowController` and have something higher-level control when flushing occurs?